vxquery-xtest/src/test/resources/Queries/XQuery/Indexing/useIndex1.xq

```diff
@@ -0,0 +1,22 @@
+(: Licensed to the Apache Software Foundation (ASF) under one
+   or more contributor license agreements.  See the NOTICE file
+   distributed with this work for additional information
+   regarding copyright ownership.  The ASF licenses this file
+   to you under the Apache License, Version 2.0 (the
+   "License"); you may not use this file except in compliance
+   with the License.  You may obtain a copy of the License at
+   
+     http://www.apache.org/licenses/LICENSE-2.0
+   
+   Unless required by applicable law or agreed to in writing,
+   software distributed under the License is distributed on an
+   "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+   KIND, either express or implied.  See the License for the
+   specific language governing permissions and limitations
+   under the License. :)
+   
+ (: Search Lucene Index :)
+ 
+for $i in collection-from-index("/Users/stevenjacobs/asterix/indexFolder", "/catalog/book")/book
```

Comment:
think of the result as you would a normal collection. It returns a collection whose top elements are named book.